claims_hosp/delphi_claims_hosp/update_indicator.py

```diff
@@ -133,14 +133,76 @@ def geo_reindex(self, data):
         data_frame.fillna(0, inplace=True)
         return data_frame
 
-    def update_indicator(self, input_filepath, outpath, logger):
+    def update_indicator_to_df(self, input_filepath, logger):
```

Comment:
question: i compared this function to update_indicator. it looks mostly the same, except for building a dataframe. any other changes?

Reply:
nope that's the only change, since the create_export_csv takes in a dataframe, but I didn't want to remove the old way until I got the approval.